lib/IRGen/GenCast.cpp

```diff
@@ -329,12 +329,12 @@ static llvm::Function *emitExistentialScalarCastFn(IRGenModule &IGM,
   }
   
   llvm::Type *returnTy = llvm::StructType::get(IGM.getLLVMContext(), returnTys);
-  
+
   auto fnTy = llvm::FunctionType::get(returnTy, argTys, /*vararg*/ false);
   auto fn = llvm::Function::Create(fnTy, llvm::GlobalValue::PrivateLinkage,
                                    llvm::Twine(name), IGM.getModule());
   fn->setAttributes(IGM.constructInitialAttributes());
-  
+
```

Comment:
I don't actually dislike these whitespace changes, but please try to avoid changing code that you're not actually changing.